vgen/src/test/java/io/gapi/vgen/testdata/discoveries/nodejs/nodejs_appengine.v1beta4.json.baseline

```diff
@@ -0,0 +1,445 @@
+
+var google = require('googleapis');
+var GoogleAuth = require('google-auth-library');
+
+var authFactory = new GoogleAuth();
+var appengine = google.appengine('v1beta4');
+
+authFactory.getApplicationDefault(function(err, authClient) {
+  if (err) {
+    console.log('Authentication failed because of ', err);
+    return;
+  }
+  if (authClient.createScopedRequired && authClient.createScopedRequired()) {
+    var scopes = ['https://www.googleapis.com/auth/cloud-platform'];
+    authClient = authClient.createScoped(scopes);
+  }
+
+  var request = {
+    // TODO: Change placeholders below to values for parameters to the 'get' method:
+
+    // Part of `name`. Name of the application to get. For example: "apps/myapp".
+    appsId: "",
+    // Auth client
+    auth: authClient
+  };
+
+  appengine.apps.get(request, function(err, result) {
+    if (err) {
+      console.log(err);
+    } else {
+      console.log(result);
+    }
+  });
+});
+var google = require('googleapis');
+var GoogleAuth = require('google-auth-library');
+
+var authFactory = new GoogleAuth();
+var appengine = google.appengine('v1beta4');
+
+authFactory.getApplicationDefault(function(err, authClient) {
+  if (err) {
+    console.log('Authentication failed because of ', err);
+    return;
+  }
+  if (authClient.createScopedRequired && authClient.createScopedRequired()) {
+    var scopes = ['https://www.googleapis.com/auth/cloud-platform'];
+    authClient = authClient.createScoped(scopes);
+  }
+
+  var request = {
+    // TODO: Change placeholders below to values for parameters to the 'delete' method:
+
+    // Part of `name`. Name of the resource requested. For example: "apps/myapp/modules/default".
+    appsId: "",
+    // Part of `name`. See documentation of `appsId`.
+    modulesId: "",
+    // Auth client
+    auth: authClient
+  };
+
+  appengine.apps.modules.delete(request, function(err, result) {
+    if (err) {
+      console.log(err);
+    } else {
+      console.log(result);
+    }
+  });
+});
+var google = require('googleapis');
+var GoogleAuth = require('google-auth-library');
+
+var authFactory = new GoogleAuth();
+var appengine = google.appengine('v1beta4');
+
+authFactory.getApplicationDefault(function(err, authClient) {
+  if (err) {
+    console.log('Authentication failed because of ', err);
+    return;
+  }
+  if (authClient.createScopedRequired && authClient.createScopedRequired()) {
+    var scopes = ['https://www.googleapis.com/auth/cloud-platform'];
+    authClient = authClient.createScoped(scopes);
+  }
+
+  var request = {
+    // TODO: Change placeholders below to values for parameters to the 'get' method:
+
+    // Part of `name`. Name of the resource requested. For example: "apps/myapp/modules/default".
```

Comment:
The `Part of`is a part of Discovery doc itself. See https://github.com/googleapis/toolkit/blob/master/vgen/src/test/java/io/gapi/vgen/testdata/discoveries/appengine.v1beta4.json#L1138 . I think we are doing the "right" thing here.